pages/venue.tsx

```diff
@@ -13,7 +14,7 @@ const VenuePage: NextPage = () => {
     <Main title="Venue" description={`About the ${conference.Name} venue.`} showHero={true}>
       <h1>Venue</h1>
       <Text>
-        {conference.Name} will be held at {conference.Venue.Name} at{' '}
+        {conference.Name} {format(conference.Date, 'y')} will be held at {conference.Venue.Name} at{' '}
```

Comment:
We've also got `conference.Instance` or something. Not sure what we use elsewhere, e.g page titles